archive/100-theorems-list/83_friendship_graphs.lean

```diff
@@ -124,10 +115,10 @@ end
   it is regular. We have shown that nonadjacent vertices of a friendship graph have the same degree,
   and if there isn't a politician, we can show this for adjacent vertices by finding a vertex
   neither is adjacent to, and then using transitivity. -/
-theorem is_regular_of_not_exists_politician (hG' : ¬exists_politician G) :
+theorem is_regular_of_not_exists_politician [nonempty V] (hG' : ¬exists_politician G) :
   ∃ (d : ℕ), G.is_regular_of_degree d :=
```

Comment:
I think I'd prefer that `[nonempty V]` would be declared once, rather than in each decl past this point in the file.

Reply:
There are a few lemmas past this point that don't use `[nonempty V]`.  I'm not against someone shuffling things around so that they can be collected in a single `variables` declaration.

Reply:
I did some shuffling; feel free to push more changes, or merge if you're happy as is.
bors d+

Reply:
Thanks!